corehq/apps/accounting/models.py

```diff
@@ -785,9 +785,13 @@ class Meta(object):
         unique_together = ('edition', 'is_trial', 'is_report_builder_enabled')
 
     @classmethod
+    @quickcache(['edition', 'is_trial', 'is_report_builder_enabled'],
+                skip_arg=lambda *args, **kwargs: not settings.ENTERPRISE_MODE or settings.UNIT_TESTING)
```

Comment:
Why skip cache for ENTERPRISE envs, line 796 `DefaultProductPlan.objects.select_related` would still be evaluated for enterprise envs, isn't it?

Reply:
I think this says the reverse: `skip if not settings.ENTERPRISE_MODE`. In other words, only cache this for enterprise envs.

Reply:
My bad, thanks